core/migrations/20220620183016_add_module_settings_column.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class AddModuleSettingsColumn extends AbstractMigration {
+
+    public function change(): void {
+        $table = $this->table('nl2_settings');
+        $table->addColumn('module', 'string', ['length' => 32, 'null' => true, 'default' => null]);
```

Comment:
Going off my previous comment, should we default to `'core'` here? Let me know your thoughts